.changeset/pink-turkeys-live.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/exchange-graphcache': patch
```

Comment:
```suggestion
'@urql/exchange-graphcache': minor
```

I'd opt for this as a feature as technically this isn't a bug. We have seen this request once before